### ISSUE TITLE
Karabiner 13.1.9 - Kernel panic at system shutdown on M1 Mac Mini / Big Sur 11.1

### ISSUE THREAD

**Issue by serge-ivamov** [pqrs-org/Karabiner-Elements#2595]
I have a stable kernel panic at shutdown with the Karabiner installed. There are no problems after uninstall.

**Comment by Maxung** [comment]
Yes, same exact system and problem. I just wondered why mine is behaving so strange and saw that in the crash report it mentioned VirtualKeyboardHID or something. After uninstalling everything works flawlessy.

**Comment by peterhop** [comment]
With Karabiner installed I get a kernel panic each time I start my Mac mini M1. I uninstalled and the problem went away. The Karabiner log showed two errors only, ‘connection was reset by a peer’. 

I installed Karabiner, then later in the day upgraded to big sur 11.1 and noticed the problem after that. I don’t remember if I restarted mu Mac with Karabiner installed before I upgraded to 11.1.

**Comment by ixunuo** [comment]
The same problem, I tried and search a lot of solution to figure out this. Finally I find it caused by Karabiner. Now I uninstall it, but my keyboard is not that easy to use now.

**Comment by pdxmph** [comment]
Same problem with pink screens at shutdown or logout on my M1 mini. Did a lot of work to isolate kexts that had come over in the migration. Removing Karabiner fixed the problem.
